docs/how-to/configure.md

```diff
@@ -57,6 +58,20 @@ Values for the *Custom* preset can be entered as values, or changed with the arr
 !!! note alert alert-primary ""
     If there is poor network connectivity between PMM Server and PMM Client, or between PMM Client and the database server being monitored, scraping every second may not be possible when the network latency is greater than 1 second.
 
+### Configure metrics resolution per-service
```

Comment:
Follow-up PR to further document this. I think things were too generic. The following PR attempts to improve the situation:
- https://github.com/percona/pmm-doc/pull/1285